packages/shared/src/models/NoTimeoutChannels.ts

```diff
@@ -0,0 +1,19 @@
+/**********************************************************************
```

Comment:
Why is it in the `src/models` folder ? could we move it to the messages folder ?